src/lib.rs

```diff
@@ -96,6 +96,71 @@
 #![cfg_attr(nightly_docs, deny(broken_intra_doc_links))]
 #![cfg_attr(nightly_docs, feature(doc_cfg))]
 
+/// Defines a mark to be used with [`hit!`] and [`chk!`].
+///
+/// # Example
+///
+/// ```
+/// #[cfg(test)]
+/// mod tests {
+///    cov_mark::def!(no_op);
+/// }
+/// ```
+#[macro_export]
+macro_rules! def {
+    ($ident:ident) => {
+        $crate::__cov_mark_private_create_mark! { static $ident }
+    };
+}
+
+/// Equivalent of [`check!`], but uses a symbol defined with [`def!`] instead
+/// of implicitly declaring one.
+///
+/// # Example
+///
+/// ```
+/// cov_mark::def!(event);
+/// #[test]
+/// fn test() {
+///     cov_mark::chk!(event);
+///     do_stuff();
+/// }
+/// # fn do_stuff() {
+/// #     cov_mark::hit!(event);
+/// # }
+/// ```
+#[macro_export]
+macro_rules! chk {
```

Comment:
I don't think either this or the next macro really need to be separate macros, as opposed to just having a variant that does not define the counter. For instance:

```rust
check!(defined $ident); // checks a pre-defined event
check!($ident); // defines the event implicitly
```

Reply:
I like this version to duplicating the calls, but I don't love the one proposed to have `define` or `defined`, so I went on with this.